resources/js/src/app/mixins/url.js

```diff
@@ -238,9 +238,13 @@ export default {
                 }
 
                 if (filter === 'price' || this.isRangeSliderFilter(attributes[filter])) {
+                    let facetInfo = this.getFacetIdInfoMap();
+
+                    let unit = (facetInfo[filter] && facetInfo[filter].unit) ? ' ' + facetInfo[filter].unit : '';
```

Comment:
The unit itself shouldn't contain the space. This should be done, when assigning the name

Reply:
Doing it like that would require us to check if the unit is not empty, otherwise spaces would be added needlessly. I think this is a cleaner solution.

Reply:
Fine, you win :D!